ha/entrypoint.c

```diff
@@ -600,6 +600,8 @@ static int ha_entrypoint_client_fom_tick(struct m0_fom *fom)
 		fop->f_opaque = ecl;
 		next_state = M0_HEC_SEND_WAIT;
 		rc = m0_rpc_post(item) == 0 ? M0_FSO_WAIT : M0_FSO_AGAIN;
+		//m0_free(req_fop_data->erf_git_rev_id.b_addr);
+		m0_buf_free(&req_fop_data->erf_git_rev_id);
```

Comment:
I don't think it is safe to free the memory at this moment.
The `m0_rpc_post()` is asynchronously.
When it returns, the request has not been finished sending. The  `req_fop_data` might be used later.
We need to call `m0_buf_free()` in `ha_entrypoint_client_fop_release()`.